ios/RNCallKeep/RNCallKeep.m

```diff
@@ -433,6 +434,19 @@ - (void)provider:(CXProvider *)provider performStartCallAction:(CXStartCallActio
     [action fulfill];
 }
 
+// Update call contact info
+RCT_EXPORT_METHOD(reportUpdatedCall:(NSString *)uuidString contactIdentifier:(NSString *)contactIdentifier)
+{
+#ifdef DEBUG
+    NSLog(@"[RNCallKeep][reportUpdatedCall] contactIdentifier = %i", contactIdentifier);
+#endif
+    NSUUID *uuid = [[NSUUID alloc] initWithUUIDString:uuidString];
+    CXCallUpdate *callUpdate = [[CXCallUpdate alloc] init];
+    callUpdate.localizedCallerName = contactIdentifier;
+
+    [self.callKitProvider reportCallWithUUID:uuid updated:callUpdate];
```

Comment:
You mean [`callKeepProvider`](https://github.com/react-native-webrtc/react-native-callkeep/blob/5cae45787f46fb772a31369c8bc89fd5167c2033/ios/RNCallKeep/RNCallKeep.m#L90) ?
I know the renaming of `callkit` to `callkeep` was a bit rough and we have `callKeep` everywhere since 😅

Reply:
Woops, yup. Fixed! Originally had this branch from the old callkit code. Thanks 👍